cli/spec/spec_helper.rb

```diff
@@ -45,22 +45,13 @@
   end
 
   config.around(:each) do |example|
-    catch :exit_with_error do
```

Comment:
Replaced specs using `expect(subject).to receive(:exit_with_error).and_throw(:exit_with_error)` with `expect{...}.to exit_with_error`, because I suspect this `catch :exit_with_error do ...` would also behave badly as regards `except` asssertions in the specs.

Reply:
Yeah it was some leftover from times before `expect{}.to exit_with_error`